src/components/MediaPlayer/VideoJS/components/js/VideoJSProgress.js

```diff
@@ -233,7 +236,7 @@ function ProgressBar({ player, handleTimeUpdate, times, options }) {
 
   player.on('timeupdate', () => {
     if (player.isDisposed()) return;
-    const curTime = player.currentTime();
+    const curTime = initCurrentTime || player.currentTime();
```

Comment:
If `initCurrentTime` is passed as a prop does it get updated somehow or does it stay static?  I'm wondering if this change blocks the progress bar from updating since it looks like `player.currentTime()` would never get returned if `initCurrentTime` is set.